src/index.js

```diff
@@ -50,6 +48,9 @@ function handlebars(options) {
     sourceMap: true
   }, options.handlebars.options);
 
+  const Handlebars = options.handlebars.module || require('handlebars');
```

Comment:
Could you add documentation for `options.handlebars.module` above the function too? Like what you wrote in the README.

Reply:
Good catch. I updated it.